models/renalStructures_CECT_segmentation/docs/README.md

```diff
@@ -0,0 +1,135 @@
+```bash
+python -m monai.bundle run download_data --meta_file configs/metadata.json --config_file "['configs/train.json', 'configs/evaluate.json']"
+```
+
+**NB**: underlying data is in LPS orientation. IF! you want to test model on your own data, reorient it from RAS to LPS with `Orientation` transform. You can see example of preprocessing pipeline in `inference.json` file of this bundle.
+
+#### **Preprocessing**
+Images are (1) croped to kidney region, all (artery,vein,excret) phases are (2) [registered](https://simpleitk.readthedocs.io/en/master/registrationOverview.html#lbl-registration-overview) with affine transform, noise removed with (3) median and (4) non-local means filter. After that, images are (5) resampled to (0.8,0.8,0.8) density and intesities are (6) scaled from [-1000,1000] to [0,1] range.
+
+## **Performance**
+On the validation subset, the values of the Dice score of the SegResNet architecture were: 0.89 for the normal parenchyma of the kidney, 0.58 for the kidney neoplasms, 0.86 for arteries, 0.80 for veins, 0.80 for ureters.
+
+When compared with the nnU-Net model, which was trained on KiTS 21 dataset, the Dice score was greater for the kidney parenchyma in SegResNet – 0.89 compared to three model variants: lowres – 0.69, fullres – 0.70, cascade – 0.69. At the same time, for the neoplasms of the parenchyma of the kidney, the Dice score was comparable: for SegResNet – 0.58, for nnU-Net fullres – 0.59; lowres and cascade had lower Dice score of 0.37 and 0.45, respectively. To reproduce, visit - https://github.com/blacky-i/nephro-segmentation
+
+
+## **Additional Usage Steps**
+
+#### Execute training:
+
+```bash
+python -m monai.bundle run training --meta_file configs/metadata.json --config_file configs/train.json
+```
+Expected result: finished, Training process started
+
+
+#### Execute training with finetuning
+```bash
+python -m monai.bundle run training --dont_finetune false --meta_file configs/metadata.json --config_file configs/train.json
+```
+Expected result: finished, Training process started, model variables are restored
+
+#### Execute validation:
+
+##### Download validation data:
+```bash
+cd models/renalStructures_CECT_segmantation
+python -m monai.bundle run download_data --meta_file configs/metadata.json --config_file "['configs/train.json', 'configs/evaluate.json']"
+```
+Expected result: finished, `data/` folder is created and filled with images.
+
+#####  Run validation script:
+```bash
+python -m monai.bundle run evaluate --meta_file configs/metadata.json --config_file "['configs/train.json', 'configs/evaluate.json']"
+```
+Expected result: finished, `Key metric: val_mean_dice best value: 0.7844631671905518 at epoch: 1` is printed.
+
+## **System Configuration**
+The model was trained for 10000 epochs on 2 RTX2080Ti GPUs with [SmartCacheDataset](https://docs.monai.io/en/stable/data.html#smartcachedataset). This takes 1 days and 2 hours, with 4 images per GPU.
```

Comment:
Could you please also add a line to show what is the minimal GPU requirements?
Reference:
https://github.com/Project-MONAI/model-zoo/tree/b245833e22544126f23925eac89da471562467a1/models/renalStructures_UNEST_segmentation#training-configuration

Reply:
Done